samples/cpp/speech_sample/speech_sample.hpp

```diff
@@ -23,10 +23,10 @@ static const char model_message[] = "Required. Path to an .xml file with a train
 
 /// @brief message for assigning calculation to device
 static const char target_device_message[] =
-    "Optional. Specify a target device to infer on. CPU, GPU, MYRIAD, GNA_AUTO, GNA_HW, "
+    "Optional. Specify a target device to infer on. CPU, GPU, MYRIAD, VPUX, GNA_AUTO, GNA_HW, "
     "GNA_HW_WITH_SW_FBACK, GNA_SW_FP32, "
     "GNA_SW_EXACT and HETERO with combination of GNA as the primary device and CPU"
-    " as a secondary (e.g. HETERO:GNA,CPU) are supported. "
+    " as a secondary (e.g. HETERO:GNA,CPU) are supported."
```

Comment:
Keep this space. There's `The sample will look for a suitable plugin for device specified.` after it

Reply:
Thank you a lot!